File: c++/1281. Subtract the Product and Sum of Digits of an Integer/1281.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        string s = to_string(n);
        int product,sum;
        product=1;
        sum=0;
        for(int i = 0; i<s.length();i++)
        {
            product*=(s[i]-'0');
            sum+=(s[i]-'0');
        }
        return product-sum;
    }
};

File: python/1281. Subtract the Product and Sum of Digits of an Integer/1281.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        s = str(n) #to iterate this int we will convert this to string
        product = 1 #product
        summ = 0 #sum 
        for i in s:
            product *= int(i) #so we just multiply each other (start with 1 cause anything * 1 = anything)
            summ += int(i)  #so we just summing each other (start with 0 cause anything + 0 = anything)
        return product - summ #retrun what you need
